MapboxNavigation/PassiveLocationManager.swift

```diff
@@ -42,6 +42,26 @@ open class PassiveLocationManager: NSObject, MGLLocationManager {
     public func requestWhenInUseAuthorization() {
         dataSource.systemLocationManager.requestWhenInUseAuthorization()
     }
+    
+    @available(iOS 14.0, *)
+    public func accuracyAuthorization() -> MBNavigationAccuracyAuthorization {
+        // CLLocationManager.accuracyAuthorization was introduced in the iOS 14 SDK in Xcode 12, so Xcode 11 doesn’t recognize it.
+        guard let accuracyAuthorizationValue = dataSource.systemLocationManager.value(forKey: "accuracyAuthorization") as? Int,
+              let accuracyAuthorization = MBNavigationAccuracyAuthorization(rawValue: accuracyAuthorizationValue) else {
+            return MBNavigationAccuracyAuthorization(rawValue: 0)!
```

Comment:
This implicitly unwrapped optional is safe and unavoidable. The raw value 0 maps to `MBNavigationAccuracyAuthorizationFullAccuracy` on iOS 10–13 and `CLAccuracyAuthorizationFullAccuracy` on iOS 14 and above.